src/pages/data-management/index.jsx

```diff
@@ -58,6 +63,9 @@ const DataManagementPage = () => {
 
     dispatch(loadProcessingSettings(activeExperimentId));
 
+    console.log(areSamplesLoaded());
+    if (!areSamplesLoaded()) dispatch(loadSamples(null, activeProjectUuid));
+
```

Comment:
after migrating to SQL experimentId = projectUuid, so the `loadSamples` function can be simplified and use only a single parameter iirc

Reply:
Yes, it worked. I removed the parameter only from the function calls because inside the function there was some api v1 code that uses it. I figured it will get removed anyways when the v1 code gets deleted.